authors/apps/article/migrations/0001_initial.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.1.3 on 2018-12-10 09:49
```

Comment:
You can remove the migrations file from the codebase as they help us avoid merge conflicts.